contracts/InterchainTokenService.sol

```diff
@@ -901,20 +869,21 @@ contract InterchainTokenService is
         bytes32 commandId,
         string calldata sourceChain,
         string calldata sourceAddress,
-        bytes memory payload,
-        bytes32 payloadHash
-    ) internal {
+        bytes calldata payload
+    ) internal override whenNotPaused {
```

Comment:
```suggestion
    ) internal override onlyRemoteService(sourceChain, sourceAddress) whenNotPaused {
```
missing modifier from `execute`. Same for `_executeWithToken`

Reply:
Nice catch, updated.